lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -130,7 +130,7 @@ public function purge()
         }
 
         $connection            = $this->em->getConnection();
-        $filterExpr            = $connection->getConfiguration()->getFilterSchemaAssetsExpression();
+        $filterExpr            = method_exists($connection->getConfiguration(), 'getFilterSchemaAssetsExpression') ? $connection->getConfiguration()->getFilterSchemaAssetsExpression() : '';
```

Comment:
the fallback should be `null` rather than the empty string, to be consistent with what DBAL 2 returns when not using the filter.

Reply:
Oops, indeed.